docs/model_server_rest_api.md

```diff
@@ -197,35 +204,40 @@ Beside numerical values, it is possible to pass binary inputs. They must be Base
   ]
 }
 ```
+
 Check [how binary data is handled in OpenVINO Model Server](./binary_input.md)
 
-Read more about *Predict API* usage examples [here](./../example_client/README.md#predict-api-1)
+Read more about [Predict API usage](./../example_client/README.md#predict-api-1)
 
 ## Config Reload API <a name="config-reload"></a>
-* Description  
+**Description**
 
 Sends requests via RESTful API to trigger config reloading and gets models and [DAGs](./dag_scheduler.md) statuses as a response.This endpoint can be used with disabled automatic config reload to ensure changes in configuration are applied in a specific time and also to get confirmation about reload operation status. Typically this option is to be used when OVMS is started with a parameter `--file_system_poll_wait_seconds 0`.
 Reload operation does not pass new configuration to OVMS server. The configuration file changes needs to be applied by the OVMS administrator. The REST API call just initiate applying the configuration file which is already present.
 
-* URL  
+**URL** 
 ```
 POST http://${REST_URL}:${REST_PORT}/v1/config/reload
 ```
-* FLOW  
+
+**Flow**
 
 Flow after receiving request:
 1) If config file was changed - reload config.
 2) If any model version directory was changed or new version was added - reload this model.
 3) If any model that is part of a DAG was changed or new version was added - reload this pipeline.
 4) In case there are no errors in the reload operation, the response includes the status of all models and DAGs, otherwise error message is returned.
```

Comment:
```suggestion
4) In case there are no errors in the reload operation, the response includes the status of all models and DAGs, otherwise the error message is returned.
```